perllib/FixMyStreet/Cobrand/Surrey.pm

```diff
@@ -46,4 +46,28 @@ sub get_town {
     return $town;
 }
 
+=head2 dashboard_export_problems_add_columns
+
+Surrey has an extra column in their stats export showing the number of subscribers to a report.
+They are set up not to subscribe the original reporter to their own report so the alert number
+is the number of users who have subscribed to the report for updates
+
+=cut
+
+sub dashboard_export_problems_add_columns {
+    my ($self, $csv) = @_;
+
+    $csv->add_csv_columns(
+        alerts => "Subscribers",
+    );
+
+    $csv->csv_extra_data(sub {
+        my $report = shift;
+
+        return {
+            alerts => $report->alerts || '0',
```

Comment:
As mentioned on Slack this should use `Admin::Reports::alerts_for_report` to get the same value as displayed in the admin report edit page.

Reply:
Just to add note from Slack discussion - this cannot use that either, because that would still be a per-row database lookup, and we can't have any per-row database lookups in this code.